Index: GamePickerWeb/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>17109488471790655
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/GamePickerWeb/obj/rider.project.model.nuget.info b/GamePickerWeb/obj/rider.project.model.nuget.info
--- a/GamePickerWeb/obj/rider.project.model.nuget.info	(revision 0c0afda39ed64425f2ad46f1df161a5312a0309d)
+++ b/GamePickerWeb/obj/rider.project.model.nuget.info	(date 1711022398884)
@@ -1,1 +1,1 @@
-17109488471790655
\ No newline at end of file
+17110223933958128
\ No newline at end of file
Index: GamePickerDataAccess/obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>17109488471226963
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/GamePickerDataAccess/obj/rider.project.model.nuget.info b/GamePickerDataAccess/obj/rider.project.model.nuget.info
--- a/GamePickerDataAccess/obj/rider.project.model.nuget.info	(revision 0c0afda39ed64425f2ad46f1df161a5312a0309d)
+++ b/GamePickerDataAccess/obj/rider.project.model.nuget.info	(date 1711022398883)
@@ -1,1 +1,1 @@
-17109488471226963
\ No newline at end of file
+17110223933834061
\ No newline at end of file
